dev/debug_osync.sh

```diff
@@ -759,7 +759,7 @@ function _PerfProfiler {												#__WITH_PARANOIA_DEBUG
 	perfString=$(ps -p $$ -o args,pid,ppid,%cpu,%mem,time,etime,state,wchan)					#__WITH_PARANOIA_DEBUG
 															#__WITH_PARANOIA_DEBUG
 	for i in $(pgrep -P $$); do											#__WITH_PARANOIA_DEBUG
-		perfString="$perfString\n"$(ps -p $i -o args,pid,ppid,%cpu,%mem,time,etime,state,wchan | :| tail -1)	#__WITH_PARANOIA_DEBUG
+		perfString="$perfString\n"$(ps -p $i -o args,pid,ppid,%cpu,%mem,time,etime,state,wchan | tail -1)	#__WITH_PARANOIA_DEBUG
```

Comment:
This is a 'all systems' wide change that can have repercussions on other environments.

Reply:
Mmh make sense... i noticed that -A is the same as -e, could that fit?  (https://man7.org/linux/man-pages/man1/ps.1.html)